drizzle-orm/src/pg-core/columns/timestamp.ts

```diff
@@ -58,12 +58,12 @@ export class PgTimestamp<T extends ColumnBaseConfig<'date', 'PgTimestamp'>> exte
 		return `timestamp${precision}${this.withTimezone ? ' with time zone' : ''}`;
 	}
 
-	override mapFromDriverValue = (value: string): Date => {
+	override mapFromDriverValue = (value: string): Date | null => {
```

Comment:
how is it possible to return null from this function

Reply:
This is a cosmetic change, since a date column can be null. I just didn't add the null to the parameter. This types don't play any role in the resulting type of the column though.